### PR TITLE
Adding Standards to Core

### PR DESCRIPTION
**Pull request by milesflo**

### CODE REVIEW

**Comment by enki-hq** [comment]
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/intro-js/console-in-javascript.md?ref=753b944c22c6b73edf865ad9c35ad75fb724845b
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/intro-js/javascript-history.md?ref=753b944c22c6b73edf865ad9c35ad75fb724845b
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/intro-js/using-javascript-on-a-web-page.md?ref=753b944c22c6b73edf865ad9c35ad75fb724845b
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/quirks-i/avoid-using-with.md?ref=753b944c22c6b73edf865ad9c35ad75fb724845b
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/quirks-i/be-careful-when-using-typeof.md?ref=753b944c22c6b73edf865ad9c35ad75fb724845b
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/quirks-i/catch-is-block-scoped.md?ref=753b944c22c6b73edf865ad9c35ad75fb724845b
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/quirks-i/changing-a-functions-context-with-fn-call-object.md?ref=753b944c22c6b73edf865ad9c35ad75fb724845b
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/quirks-i/check-if-a-function-exists.md?ref=753b944c22c6b73edf865ad9c35ad75fb724845b
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/quirks-ii/inaccuracy-of-binary-floating-point-format.md?ref=753b944c22c6b73edf865ad9c35ad75fb724845b
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/quirks-ii/nan-is-a-number.md?ref=753b944c22c6b73edf865ad9c35ad75fb724845b
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/quirks-ii/the-difference-between-null-undefined-and-nan.md?ref=753b944c22c6b73edf865ad9c35ad75fb724845b
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/quirks-ii/truthiness.md?ref=753b944c22c6b73edf865ad9c35ad75fb724845b
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/quirks-ii/weak-vs-strict-equality-operator.md?ref=753b944c22c6b73edf865ad9c35ad75fb724845b
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/recipes-i/easily-generate-a-random-hex-color.md?ref=753b944c22c6b73edf865ad9c35ad75fb724845b
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/recipes-i/get-a-random-item-from-an-array.md?ref=753b944c22c6b73edf865ad9c35ad75fb724845b
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/recipes-i/setting-default-values-with-short-circuiting.md?ref=753b944c22c6b73edf865ad9c35ad75fb724845b
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/recipes-i/use-the-apply-function-to-get-the-min-or-max-of-an-array.md?ref=753b944c22c6b73edf865ad9c35ad75fb724845b
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/recipes-i/using-test-with-regexp.md?ref=753b944c22c6b73edf865ad9c35ad75fb724845b
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/recipes-ii/counting-the-months-from-zero.md?ref=753b944c22c6b73edf865ad9c35ad75fb724845b
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/recipes-ii/format-currency-in-javascript.md?ref=753b944c22c6b73edf865ad9c35ad75fb724845b
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/recipes-ii/milliseconds-since-epoch.md?ref=753b944c22c6b73edf865ad9c35ad75fb724845b
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/recipes-ii/performing-a-function-at-timed-intervals.md?ref=753b944c22c6b73edf865ad9c35ad75fb724845b
- [ ] https://enkipro.com/preview-insight/sagelabs/content/JavaScript/Core/recipes-ii/round-numbers-to-n-decimals.md?ref=753b944c22c6b73edf865ad9c35ad75fb724845b

File: JavaScript/Core/check-if-an-argument-is-a-number.md [hunk 1]
```diff
@@ -67,10 +67,10 @@ function isNumber(n){
...
-*`typeof`
+* `isNaN`
+* `parseFloat`
+* `isFinite`
+* `isInfinite`
+* `typeof`
 * `isNum`
-*`isValud`
\ No newline at end of file
+* `isValud`
```
Comment (lizTheDeveloper) [inline JavaScript/Core/check-if-an-argument-is-a-number.md]: js.evaluate-expressions

File: JavaScript/Core/intro-js/console-in-javascript.md [hunk 2]
```diff
@@ -11,6 +11,9 @@ type: normal
 
 category: must-know
 
+standard: 
+  js.javascript-syntax
```
Comment (lizTheDeveloper) [inline JavaScript/Core/intro-js/console-in-javascript.md]: js.standard-library

File: JavaScript/Core/quirks-i/avoid-using-with.md [hunk 3]
```diff
@@ -13,6 +13,9 @@ type: normal
 
 category: best practice
 
+standard: 
+  js.evaluate-expressions: 10
```
Comment (lizTheDeveloper) [inline JavaScript/Core/quirks-i/avoid-using-with.md]: js.control-flow

File: JavaScript/Core/quirks-i/catch-is-block-scoped.md [hunk 4]
```diff
@@ -9,6 +9,9 @@ type: normal
 
 category: caveats
 
+standard: 
+  js.evaluate-expressions: 10
```
Comment (lizTheDeveloper) [inline JavaScript/Core/quirks-i/catch-is-block-scoped.md]: js.control-flow

File: JavaScript/Core/quirks-ii/weak-vs-strict-equality-operator.md [hunk 5]
```diff
@@ -9,6 +9,9 @@ type: normal
 
 category: must-know
 
+standard: 
+  js.evaluate-expressions: 20
```
Comment (lizTheDeveloper) [inline JavaScript/Core/quirks-ii/weak-vs-strict-equality-operator.md]: also js.write-expressions

File: JavaScript/Core/quirks-ii/truthiness.md [hunk 6]
```diff
@@ -13,6 +13,9 @@ type: normal
 
 category: tip
 
+standard: 
+  js.evaluate-expressions: 10
```
Comment (lizTheDeveloper) [inline JavaScript/Core/quirks-ii/truthiness.md]: also js.write-expressions

File: JavaScript/Core/recipes-i/get-a-random-item-from-an-array.md [hunk 7]
```diff
@@ -11,6 +11,8 @@ type: normal
 
 category: how to
 
+standard: js.data-types-structures
+
```
Comment (lizTheDeveloper) [inline JavaScript/Core/recipes-i/get-a-random-item-from-an-array.md]: js.write-expressions

Comment (lizTheDeveloper) [reply JavaScript/Core/recipes-i/get-a-random-item-from-an-array.md]: as well

File: JavaScript/Core/recipes-i/setting-default-values-with-short-circuiting.md [hunk 8]
```diff
@@ -11,6 +11,9 @@ type: normal
 
 category: pattern
 
+standard:
+  js.write-expressions: 10
```
Comment (lizTheDeveloper) [inline JavaScript/Core/recipes-i/setting-default-values-with-short-circuiting.md]: also evaluate

File: JavaScript/Core/recipes-i/using-test-with-regexp.md [hunk 9]
```diff
@@ -11,6 +11,9 @@ type: normal
 
 category: feature
 
+standard:
+  js.write-expressions: 10
```
Comment (lizTheDeveloper) [inline JavaScript/Core/recipes-i/using-test-with-regexp.md]: data structures and their respective ops too

File: JavaScript/Core/recipes-ii/counting-the-months-from-zero.md [hunk 10]
```diff
@@ -13,6 +13,9 @@ type: normal
 
 category: caveats
 
+standard: 
+  js.write-expressions: 10
+
```
Comment (lizTheDeveloper) [inline JavaScript/Core/recipes-ii/counting-the-months-from-zero.md]: also evaluate

**Comment by lizTheDeveloper** [comment]
merged via cli, I fixed these things

**Pull request closed without merging**